src/pymortests/algorithms/symplectic.py

```diff
@@ -0,0 +1,73 @@
+
+import numpy as np
+from pymor.algorithms.symplectic import (psd_complex_svd, psd_cotengent_lift,
+                                         psd_svd_like_decomp,
+                                         symplectic_gram_schmidt)
+from pymor.operators.symplectic import CanonicalSymplecticFormOperator
+from pymor.vectorarrays.numpy import NumpyVectorSpace
+
+
+def test_symplecticity():
+    """Check symplecticity of symplectic basis generation methods."""
+    basis_gen_methods = [psd_cotengent_lift, psd_complex_svd, psd_svd_like_decomp]
```

Comment:
It's better to let `basis_gen_methods` be a parameter and use `@pytest.mark.parametrize`. If you don't know how, I can do it.